src/base/file_util.h

```diff
@@ -328,6 +344,56 @@ inline static int HardLinkDir(const std::string& src, const std::string& dest) {
     return 0;
 }
 
+// list of paths of all files under the directory 'dir' when the filename matches the pattern
+// do not include any subfolder
+inline static std::vector<std::string> FindFiles(const std::string& dir, const std::string& pattern) {
+    // convert ls pattern `test*` to regex pattern `test.*`
+    std::string converted_pattern;
+    for (auto c : pattern) {
```

Comment:
`glob`

Reply:
c glob seems not available for windows.

Reply:
yes. it is POSIX. anyway most of methods in file_util.h is linux-specific.